astroid/node_classes.py

```diff
@@ -4910,21 +4910,20 @@ def get_children(self) -> typing.Generator["PatternTypes", None, None]:
             yield from self.patterns
 
 
-class MatchMapping(NodeNG):
+class MatchMapping(mixins.AssignTypeMixin, NodeNG):
```

Comment:
I was wondering if it could be interesting to add an example of such node in the docstring. What do you think about it?

Reply:
You got me 😄 I've been putting off doing that.
Added a new commit with examples for each `Match` node.

The link to the docs preview:
https://pylint--1010.org.readthedocs.build/projects/astroid/en/1010/api/astroid.nodes.html#astroid.nodes.Match

Reply:
Nice job! At least i have been a bit useful! :wink:

Reply:
A review is always useful!